src/frontend/src/players/pipelines/ffmpegPlayingPipeline.ts

```diff
@@ -171,7 +177,7 @@ class PlayingPipeline extends StateMachineImpl<PipelineState> {
 
   kill(): void {
     this.command?.unpipe(this.streamer)
-    this.command?.kill('SIGKILL')
```

Comment:
Are you sure it's a good idea?
I'm afraid, we'll have the same issue as before, the process isn't killed in Linux or Windows, and sometimes we have to specify the Kill level.

Reply:
"this.command" is not an FfmpegCommand anymore. It has simply be moved here -> https://github.com/taktik/flowr-desktop/pull/63/files/b676c187bac3b9fbe4aa3dad43a540082b3543e6#diff-e849dcaa6e5d73a418062757f27427a77e3786fd950a6d0ef6f72199e9daa1b9R81

Reply:
my bad :)